modules/Core/src/main/java/org/terasology/rendering/nui/layers/ingame/inventory/InventoryScreen.java

```diff
@@ -43,4 +61,103 @@ public EntityRef get() {
     public boolean isModal() {
         return false;
     }
+
+    /*
+      The numbersBetween() and getTransferEntity() methods were
+      originally in the InventoryCell class. They were copied over
+      to here because they are private functions that the
+      moveItemSmartly() method needs to function. The first section
+      of code in onClosed() is based on the moveItemSmartly() method.
+    */
+    private List<Integer> numbersBetween(int start, int exclusiveEnd) {
```

Comment:
Incidentally now that we seem to have this method in three different places it might be time to add it to https://github.com/MovingBlocks/TeraMath instead as a static method then just use it from there. Don't worry about that for this PR (gets a little more complicated to involve a lib update) but could be an idea for what to try next :-)